ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -177,6 +179,31 @@ class H2OGridSearch(override val uid: String)
       "The fit method of the grid search must be called first to be able to obtain a list of models.")
   }
 
+  private def getMetricValue(model: H2OMOJOModel, metric: H2OMetric): Double = metric match {
+    case H2OMetric.GLRMMetric =>
+      val ast = parse(model.getModelDetails())
+
+      def getColumnName(value: JValue): String = {
```

Comment:
WDYT of extracting the part for obtaininig GLRM metric to a different method so the case is smaller?

Reply:
np, will do